packages/sit-onyx/src/components/examples/GridPlayground/EditGridElementDialog/EditGridElementDialog.vue

```diff
@@ -68,62 +68,46 @@ const handleCheckboxChange = (isChecked: boolean, breakpoint: OnyxBreakpoint) =>
       <div class="dialog__header">
         <OnyxHeadline is="h2">{{ label }}</OnyxHeadline>
 
-        <OnyxCheckbox v-model="state.isFullWidth" label="Full Width" value="is-full-width" />
```

Comment:
Please do not remove this feature. Even though it is not in Figma, we want to support it
```suggestion
        <OnyxCheckbox v-model="state.isFullWidth" label="Full Width" value="is-full-width" />
```